src/test/java/com/iexec/worker/compute/ComputeManagerServiceTests.java

```diff
@@ -111,9 +100,24 @@ void beforeEach() {
         MockitoAnnotations.openMocks(this);
     }
 
+    private TaskDescription createTaskDescription(boolean isTeeTask) {
```

Comment:
Any reason why you don't use the same "builder" pattern as in the other classes (=providing a static builder which tests can complete and build themselves)?

Reply:
I used the "builder" pattern when tests required it.
It was not the case in this tests class, so I did nothing fancy

Reply:
What about [this line](https://github.com/iExecBlockchainComputing/iexec-worker/blob/e1c59b8f732e880cb1ede9bc798ed53290728903/src/test/java/com/iexec/worker/compute/ComputeManagerServiceTests.java#L143-L145)? (There are a few others, similar to this one.)
In other files, you use this pattern to override fields. Such as [here](https://github.com/iExecBlockchainComputing/iexec-worker/blob/e1c59b8f732e880cb1ede9bc798ed53290728903/src/test/java/com/iexec/worker/compute/app/AppComputeServiceTests.java#L98-L100).